models/POP/shell_scripts/cesm2_1/CESM_DART_config

```diff
@@ -0,0 +1,317 @@
+#!/bin/csh
+#
+# Copyright 2020 University Corporation for Atmospheric Research
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License.
+# Please view the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# ==============================================================================
+#
+# ---------------------
+# Purpose
+# ---------------------
+#
+# This script integrates DART with a pre-existing CESM multi-instance case.
+# It must be run from a valid CASEROOT directory. If the case was created
+# using one of the DART scripts, this script should be staged in the
+# CASEROOT directory automatically, and DARTROOT is set at that time.
+#
+# POP is the only active model component.
+# CESM starts and stops to allow for POP to assimilate every 24 hours.
+#
+# This script will build the DART executables if they are not found.
+#
+# There are many CESM binary files in big-endian format, and DART reads
+# some of them, so you MUST compile DART accordingly e.g.,
+# ifort -convert big_endian
+# Contact dart@ucar.edu if you want to use another compiler.
+#
+# ---------------------
+# How to set up the script
+# ---------------------
+#
+# -- Ensure DARTROOT references a valid DART directory.
+# -- Examine the whole script to identify things to change for your experiments.
+# -- Provide any initial files needed by your run:
+#       inflation
+#       sampling error correction
+# -- Run this script.
+# -- Edit the DART input.nml that appears in the ${CASEROOT} directory.
+# -- Submit the job using ${CASEROOT}/case.submit
+#
+# ==============================================================================
+# Get the environment of the case - defines number of instances/ensemble size ...
+# Each model component has their own number of instances.
+# ==============================================================================
+
+setenv cesmtag              cesm2_1_1
+# Trim the minor version off cesmtag to get the major release
+setenv cesmtagmajor `echo ${cesmtag} | cut -c1-7`
```

Comment:
I think there is reason to source the DART_params.csh here instead of setting the cesmtag, etc.
Furthermore, line 112 requires a DARTROOT environment varriable - which is only set in DART_params.csh